basyx.common/basyx.authorization/src/main/java/org/eclipse/digitaltwin/basyx/authorization/CommonSecurityConfiguration.java

```diff
@@ -49,6 +49,10 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 				.authorizeHttpRequests(authorize -> authorize
 						.requestMatchers("/actuator/health/**").permitAll()
 						.requestMatchers(HttpMethod.OPTIONS, "/**").permitAll()
+						.requestMatchers("/swagger-ui/**").permitAll()
```

Comment:
Add a configuration option to enable or disable the swagger UI while in Authorization.